src/publisher/admin.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
-from . import models
+from . import models, aws_events
```

Comment:
I think there is (or was?) a module called events already for something else. It was once called sns_events or something similar. when we get different types of events that need to be sent out, I'll deal with the  refactor then. It's pretty benign right now ;)